indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -3660,6 +3728,9 @@ protected Map<PartitionIdType, SequenceOffsetType> getHighestCurrentOffsets()
                 Entry::getValue,
                 (v1, v2) -> makeSequenceNumber(v1).compareTo(makeSequenceNumber(v2)) > 0 ? v1 : v2
             ));
+
+        partitionIds.forEach(partitionId -> currentOffsets.putIfAbsent(partitionId, offsetsFromMetadataStorage.get(partitionId)));
+        return currentOffsets;
       }
       // nothing is running but we are not suspended, so lets just hang out in case we get called while things start up
       return ImmutableMap.of();
```

Comment:
We need to return offsetsFromMetadataStorage here as well since the lag is computed differently now

Reply:
Thanks Amatya, Addressed your comments in the latest commit.